---
title: "Exploring the Interplay Between Fibre Intake, Exercise, and Gut Microbiota in Modulating Cardiovascular Health in a Westernization Context"
author: "Rui Xiang Yu, Houria Afshar Moghaddam, Brooke Macleod, Quinlan Torstensen"
number-sections: true
format: 
    html:
        toc: true
        toc-depth: 3
        auto-stretch: false
        embed-resources: true
    pdf:
        toc: true
        toc-depth: 3 
        fig-pos: "H"
        auto-stretch: false
editor: source
bibliography: references.bib
execute:
    echo: false
    warning: false
---

```{r}
library(tidyverse)
library(knitr)
```

## Introduction

This report contains the methods and results generated throughout the making of this project. Our research question is:

*In a westernization context, how do dietary fibre intake and exercise affect cardiometabolic health, and does this associate with unique microbiome profiles?*

Our aims are:

1.  Investigate the optimal predictors for cardiovascular health, such as dietary fibre and exercise.
2.  Compare diversity metrics of groups with adequate and inadequate dietary fibre intake, and high and low exercise levels in relation to cardiometabolic health.
3.  Identify taxa that are differentially abundant and co-occur among the different fibre intake and exercise groups, as well as cardiovascular condition.

## Methods

### Data processing
The dataset was sourced from de la Cuesta-Zuluaga et al. 2018’s paper, whose raw DNA FASTQ files can be found at the SRA-NCBI under BioProject PRJNA417579 (@de2018gut). The dataset comprises the sequences corresponding to the V4 hypervariable region of the 16S rRNA gene, collected in 2014 (@de2018gut). The primers used were F515 and R806 and sequencing was done with Illumina MiSeq (@de2018gut). There are 441 samples of men and women, across a wide range of ages and body mass indices, that lived in different Colombian cities (Bogotá, Medellín, Cali, Barranquilla, Bucaramanga). None of the participants were underweight (Body Mass Index <18.5 kg/m²), pregnant, consumed antibiotics or antiparasitics three months before sample collection, with cancer, with gastrointestinal diseases, or with neurodegenerative diseases (@de2018gut).

The accompanying metadata for each patient consists of multiple demographic parameters (age, biological sex, city of residence), anthropometric measures (body mass index, body fat percentage, waist circumference), lipid profiles in blood (HDL, LDL, adiponectin), macronutrient consumption, blood pressure, glucose metabolism profiles, and stool consistencies. A full list of available metadata with each variable’s range or factors can be found in @tbl-metadata-ranges.


```{r}
#| label: tbl-metadata-ranges
#| tbl-cap: Table of the multiple metadata information available for each patient. For categorical variables, it contains all factors. For numerical variables, it contains the numerical range of available values.

metadata_ranges <- read_tsv("../results/0-metadata_summary.tsv")
knitr::kable(metadata_ranges)

```

The data was processed with QIIME2 version 2025.4.0 (@bolyen2019reproducible). The script to import the data and demultiplex the sequences can be found in `bin/01-qiime2_data_processing.sh`. The maximum read length was 251 nucleotides (nts). Based on a random subsample of 10,000 reads, 98% of reads have a read length of 251 nts, while 2% have a read length of 250 nts. The maximum number of reads in a sample was 117,562, whereas the minimum number was 4,305. The mean number of reads was 40,657.89.

The script for filtering and downstream QIIME2 processes are in `bin/02-qiime2_data_filtering.sh`. The denoising tool chosen was DADA2 (@callahan10high). The Phred quality score threshold chosen was 30. The base pair at position 251’s median quality was 29. A truncation length of 250 was chosen. After denoising, all 441 samples were still retained. The maximum number of reads in a sample changed to 106,116, whereas the minimum number changed to 3,665.

With QIIME2’s taxonomic classifier, amplicon sequence variants (ASVs) were classified with a pre-trained Naïve Bayes model built on the SILVA version 138 99% OTUs, trained for the primer pair 515F/806R, which targets the V4 region of the 16S rRNA gene. A phylogenetic tree was subsequently generated by aligning multiple sequences with MAFFT, masking hypervariable regions, constructing an approximately maximum-likelihood tree with FastTree2, and applying midpoint rooting to obtain a rooted phylogeny for downstream analyses.

Afterwards, sequences corresponding to mitochondria or chloroplasts, as well as non-bacterial sequences were filtered out. Based on the subsequently generated rarefaction curve (@fig-rarefaction), ASVs saturate at around ~10,000 reads for most samples. A cutoff of 24,406 reads was chosen as this retained 79.49% of the original number of features (8,705 out of 10,951) and 333 samples. Thus, 108 samples were discarded. No batch correction was applied to the data as de la Cuesta-Zuluaga et al. found no significant difference across runs and an internal control was also present (@de2018gut).

![Alpha rarefaction plot where each line represents a sample. The dotted red line represents a sequencing depth of 24,406 reads.](../results/06-rarefaction_curve.png){#fig-rarefaction}

### Exploratory Data Analyses and Preliminary Analyses
Prior to rarefying, using all 441 samples, a redundancy analysis (RDA) was performed. The OTU table was aggregated to the phylum level and a Hellinger transformation was applied. The transformed and aggregated OTU table was used as the response variable and the entire metadata was used as the predictors. An ANOVA Permutation test was run on the model to determine significance of the covariates. Results can be found in @tbl-prelim-rda. We can see that fibre appears to be significantly associated to the OTU table.

```{r}
#| label: tbl-prelim-rda
#| tbl-cap: Redundancy Analysis results with hellinger-transformed phylum counts as a response matrix. Contains for each variable its degrees of freedom (Df), variance explained by the variable (Variance), its F-statistic (F), and the p-value from ANOVA-like permutation tests (Pr(>F)).


prelim_rda <- read_tsv("../results/04-prelim_anova_rda.tsv")
knitr::kable(prelim_rda)

```

Furthermore, a logistic regression model was fit onto the metadata, where cardiovascular status was the response variable. Model results are in @tbl-prelim-logreg. We can see that MET units are significantly associated with cardiovascular status.

```{r}
#| label: tbl-prelim-logreg
#| tbl-cap: Logistic regression results with cardiometabolic status as the response variable. Contains for each variable the log-odds coefficient (Estimate), the standard error of the coefficient (Std. Error), its z-statistic (z), and its p-value (Pr(>|z|)). 


prelim_logreg <- read_tsv("../results/05-log_reg_summary.tsv")
knitr::kable(prelim_logreg)

```

The distributions of the variables of interest did not change considerably after rarefying. The distribution metabolic equivalent of task (MET) was heavily right-skewed (@fig-hist_rare). The distribution of daily fibre intake was bell-shaped (@fig-hist_rare). The number of female and male participants, as well as participants with a healthy or abnormal cardiometabolic status, are balanced throughout (@fig-hist_rare). The number of participants with healthy cardiometabolic status went from 269 to 204, while those with an abnormal status went from 172 to 129. The number of female participants went from 229 to 178, while male participants went from 212 to 155. The distribution of age in years was also explored and it was roughly bell-shaped, almost uniform-like (@fig-hist_rare). The distributions of these variables prior to filtering (all 441 initial samples portrayed) can be found in @fig-hist_all.

![Histogram distributions of the 333 participants’ fibre intake (left), metabolic equivalent of tasks (middle), and age (right).  Distributions are colored by biological sex (top) and cardiometabolic status (bottom). Dotted vertical lines represent the median for each sub-group.](../results/07-fiber_met_age_hists_after_rarefaction.png){#fig-hist_rare}

![Histogram distributions of the 441 participants’ fibre intake (left), metabolic equivalent of tasks (middle), and age (right).  Distributions are colored by biological sex (top) and cardiometabolic status (bottom). Dotted vertical lines represent the median for each sub-group.](../results/01-fiber_met_age_hists.png){#fig-hist_all}

As mentioned previously, participants will be divided into two fibre intake and exercise level groups, while considering cardiovascular status as well. High fibre intake is considered as 20 grams of fibre or above. High exercise level is considered as more than 1000 metabolic equivalent of task minutes per week. These cut-offs were considered based in the distributions observed as well as guidelines on the adequate number of MET units and fibre intake. The sample sizes of each sub-group are reported in @tbl-cv_fibre_exercise_counts. The smallest groups, representing participants with high fibre intake and low exercise, included 8 with a healthy and 9 with an abnormal cardiovascular status. Given these small sample sizes, all 333 samples were retained for downstream analyses, with no further filtering performed. Based on the observed distributions and lack of clear outliers, this approach was considered valid.

```{r}
#| label: tbl-cv_fibre_exercise_counts
#| tbl-cap: Distribution of Participants by Cardiometabolic Health Status, Fibre Intake, and Exercise Level. High fibre intake is 20 grams or more of daily fibre. High exercise level is more than 1000 MET minutes per week.

group_counts <- read_tsv("../results/08-cvstatus_fibre_exercise_counts.tsv")
knitr::kable(group_counts)
```

### Aim 1: Multiple Linear Regression

## References
